test/metabase/test/util/log.clj

```diff
@@ -195,9 +202,11 @@
         logger        (exact-ns-logger a-namespace)]
     (try
       (.addAppender logger appender (->Level level) nil)
+      ;; prevent events to be passed to the parent logger's appenders
+      ;; https://logging.apache.org/log4j/2.x/manual/configuration.html#Additivity
```

Comment:
Is this comment intentional here?

Reply:
oh I forgot to remove it. I initially setAdditiviy here, but then decided to move it to `do-with-log-level`.

Will remove this comment